services/121-service/src/instance/instance.controller.ts

```diff
@@ -49,20 +47,4 @@ export class InstanceController {
   ): Promise<InstanceEntity> {
     return await this.instanceService.updateInstance(updateInstanceDto);
   }
-
-  @AuthenticatedUser({ isAdmin: true })
-  @ApiOperation({ summary: 'Update instance monitoring question' })
-  @ApiResponse({
-    status: HttpStatus.NOT_FOUND,
-    description: 'No instance found',
-    type: InstanceEntity,
-  })
-  @Patch('monitoringQuestion')
-  public async updateMonitoringQuestion(
```

Comment:
If there are no more "Monitoring questions" updated/added, can we remove ALL "Monitoring-question"-features?

So also the related "exports", the storing of the answers, etc?

@RubenGeo Would you know?

(Might not be necessary to include in the PR, but could be added to the "collateral removals"-list.)

Reply:
@PeterSmallenbroek actually mentioned the same to me before realising you were already looking at this. I am going to address this in AB#26403